### ISSUE TITLE
Numpy non-tuple nd-indexing deprecation warning in UFLACS

### ISSUE THREAD

**Issue by w1th0utnam3** [FEniCS/ffcx#53]
Since Numpy 1.15.0 the following warning is raised in UFLACS:
```
FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
```
caused by https://github.com/FEniCS/ffcx/blob/98fa5b36a355a43e0019618ef46196dd932ab1e0/ffc/uflacs/analysis/dependencies.py#L50
Can be reproduced using NumPy 1.15.0 and ufl file:
```
el = FiniteElement("Lagrange", triangle, 1)
v = TestFunction(el)
u = TrialFunction(el)
a = inner(grad(u), grad(v)) * dx
```

Warning was introduced in NumPy here: https://github.com/numpy/numpy/pull/9686

https://github.com/numpy/numpy/issues/11411 suggests that if the warning is issued, it is always ok to replace to just transform the argument in a tuple but I'm not sure if the UFLACS code always causes this warning. Don't have the time at the moment to identify what inputs this UFLACS funtion receives.

**Comment by blechta** [comment]
Is this fixed by #54?

**Comment by w1th0utnam3** [comment]
Yes

**Comment by w1th0utnam3** [comment]
Well at least it removes the warning, I didn't check whether it's 100% correct with respect to possible arguments to the function.

**Comment by blechta** [comment]
Fixed by #54.
